README.md

```diff
@@ -56,6 +56,26 @@ kellerkinder.additionalPhpExtensions = [
 ];
 ```
 
+### kellerkinder.additionalCaddyServerAlias
+Define additional server aliases that are added to caddy.
+
+*_Example_*
+```
+kellerkinder.additionalCaddyServerAlias = [
```

Comment:
We can omit "Caddy" in the options, since we don't offer any other webservers in our environment 🙂

Reply:
dont :)